GitPrompt.ps1

```diff
@@ -82,6 +82,9 @@ $global:GitPromptSettings = New-Object PSObject -Property @{
 
     AutoRefreshIndex                            = $true
 
+    # Valid values are "Full", "Compact", and "Minimal"

```

Comment:
Unfortunately, that doesn't work because it's a set of properties being initialized, not a variable.  We could move it outside the GitPromptSettings declaration as a special case, but that's kind of ugly.